neuralmonkey/decoding_function.py

```diff
@@ -1,59 +1,93 @@
 import tensorflow as tf
 
-def attention_decoder(decoder_inputs, initial_state, attention_objects,
-                      embedding_size, cell, output_size=None,
-                      loop_function=None, dtype=tf.float32, scope=None):
+from neuralmonkey.logging import debug
 
-    if output_size is None:
-        output_size = cell.output_size
+def attention_decoder(decoder_inputs, initial_state, attention_objects,
```

Comment:
Hm, to asi jo. Ale dekodér bez attentionu je dneska skoro kuriozita, ne?

Tím chci říct: jestli se ti to chce udělat, tak směle do toho, je to určitě krok k většímu ordnungu, ale mně by se to moc dělat nechtělo.